### PR TITLE
*: fix ci

### PR DESCRIPTION
**Pull request by tiancaiamao**

### CODE REVIEW

File: util/types/time.go [hunk 1]
```diff
@@ -1202,16 +1202,14 @@ func checkTimestampType(t TimeInternal) error {
...
-		log.Infof("checkTimestampType failed, t=%v", t)
-		return errors.Trace(err)
+	if compareTime(t, maxTimestamp) > 0 || compareTime(t, minTimestamp) < 0 {
+		return errors.Trace(ErrInvalidTimeFormat)
 	}
 
-	if t1.After(maxTimestamp) || t1.Before(minTimestamp) {
-		return ErrInvalidTimeFormat
+	if _, err := t.GoTime(gotime.Local); err != nil {
+		return errors.Trace(ErrInvalidTimeFormat)
```
Comment (zimulala) [inline util/types/time.go]: It‘s better to use `err`.

File: executor/distsql.go [hunk 2]
```diff
@@ -465,7 +465,7 @@ func (e *XSelectIndexExec) nextForSingleRead() (*Row, error) {
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		err = decodeRawValues(values, schema, e.ctx.GetSessionVars().GetTimeZone())
+		err = decodeRawValues(values, schema, time.UTC)
```
Comment (shenli) [inline executor/distsql.go]: Add comment for this.

File: util/types/time.go (hunk 1 above)
Comment (shenli) [inline util/types/time.go]: Why not output err message?

File: util/types/time.go [hunk 3]
```diff
@@ -1202,16 +1202,14 @@ func checkTimestampType(t TimeInternal) error {
...
-	if err != nil {
-		log.Infof("checkTimestampType failed, t=%v", t)
-		return errors.Trace(err)
+	if compareTime(t, maxTimestamp) > 0 || compareTime(t, minTimestamp) < 0 {
+		return errors.Trace(ErrInvalidTimeFormat)
 	}
 
-	if t1.After(maxTimestamp) || t1.Before(minTimestamp) {
-		return ErrInvalidTimeFormat
+	if _, err := t.GoTime(gotime.Local); err != nil {
```
Comment (shenli) [inline util/types/time.go]: It is safe to use gotime.Local here?

Comment (tiancaiamao) [inline util/types/time.go]: Yes.
This is just a test, the converted value is never used.
If `t` can be converted to go time, it should be a valid time format.

**Comment by tiancaiamao** [comment]
PTAL @shenli @zimulala

**Comment by zimulala** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**